pages/docs/dragonite/config.mdx

```diff
@@ -50,14 +50,29 @@ enable = true
 # token_init_enabled = true
 # don't rely on remote auth and only use accounts with stored token - true by default
 # token_refresh_only = true
+
+[auth.nk]
+enable = false
+#login_delay = 20
+#token_init_enabled=true
+#token_refresh_only = true
 ```
 
-⚠ Additional config variables, but only put them if you know what you do ⚠
+⚠ Additional config variables.
+Do **NOT** change these unless you really know what you are doing! ⚠
 | Key | Default | Description |
 | :-- | :-- | :-- |
-| token_refresh_concurrency | 10 | define concurrency for the token background refresher |
-| token_init_concurrency | 1 | define concurrency for the token background initer |
-| token_init_interval_seconds | PTC: 1, NK: 180 | define the interval to request tokens |
+| `token_refresh_concurrency` | 10 | define concurrency for the token background refresher |
+| `token_init_concurrency` | 1 | define concurrency for the token background initer |
+| `token_init_interval_seconds` | PTC: 1, NK: 180 | define the interval to request tokens |
+| `token_init_start_delay_minutes` | 5 | define a delay for starting the token initer |
+| `remote_auth_timeout_seconds` | 60 | define a timeout for remote auth requests |
+| `remote_init_retry_delay_minutes` |  | define a delay for retrying remote auth requests |
+| `remote_auth_secret` | "" | define a secret for remote auth requests if needed |
+| `token_refresh_days` | 28 | define the days after which a token should be refreshed |
```

Comment:
Hmm that's different in the NK case, maybe mention? Could confuse nk people?! :)